packages/medusa/src/api/routes/admin/returns/list-returns.ts

```diff
@@ -36,3 +44,18 @@ export default async (req, res) => {
 
   res.json({ returns, count: returns.length, offset, limit })
 }
+
+export type AdminGetReturnsResponse = PaginatedResponse & {
+  returns: Return[]
+}
+
```

Comment:
This should be moved to `api/routes/admin/returns/index.ts` and should be named `AdminReturnsListRes`